test/host/test_all.py

```diff
@@ -1,4 +1,21 @@
 #!/usr/bin/env python3
```

Comment:
remove this, we don't know where the users python is installed. Might want to check for python version like this: https://stackoverflow.com/questions/1093322/how-do-i-check-what-version-of-python-is-running-my-script